packages/plugin-vue-jsx/index.js

```diff
@@ -55,9 +55,8 @@ function vueJsxPlugin(options = {}) {
           include: /\.ts$/
         },
         define: {
-          __VUE_OPTIONS_API__: true,
-          __VUE_PROD_DEVTOOLS__: false,
-          ...config.define
+          __VUE_OPTIONS_API__: true || config.define?.__VUE_OPTIONS_API__,
+          __VUE_PROD_DEVTOOLS__: false || config.define?.__VUE_PROD_DEVTOOLS__
```

Comment:
`false || config.define?.__VUE_PROD_DEVTOOLS__` This would **always** fallback to the rhs :thinking:

Reply:
😅 sorry about that.